content/docs/card/feature-card.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Feature Card
+description: When the cursor hovers over the button, the card skews and the check mark appears.
+author: PrithwiHegde
+---
+
+<ComponentPreview name="card-feature-card--docs" />
```

Comment:
@Prithwi32 You need to rename this file as well to `subscribe-card.mdx` and update the `feature-card` in the docs to `subscribe-card`.